packages/framework/framework/src/useTokens.ts

```diff
@@ -5,6 +5,7 @@ import { TokenSettings } from './useStyling';
 
 export { applyTokenLayers, applyPropsToTokens, customizable, patchTokens } from '@fluentui-react-native/use-tokens';
 
+// A hook function to build a set of tokens from a passed in theme as well as a cache object
```

Comment:
Is it possible to add an example of how this might be invoked?

Reply:
I tried. It gets really big and wordy for VS Code. I think that would be better with a README.md somewhere. I'll look into that.

Reply:
I think I'll leave it as a follow-up. Any README with info on compressible should also have info on stagedComponent and compose. That becomes a bigger task than the simple purpose of this PR.